tests/test_openclip_vqa.py

```diff
@@ -0,0 +1,73 @@
+import json
+import os
+import re
+import subprocess
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import pytest
+
+from .test_examples import TIME_PERF_FACTOR
+
+
+if os.environ.get("GAUDI2_CI", "0") == "1":
+    # Gaudi2 CI baselines
+    MODELS_TO_TEST = {
```

Comment:
Tests take 49.57s to run , if format is acceptable, need to update Makefile as well.

Reply:
on my side, the test on a new container where data is not present, this takes 3.5min. 
Likely would be an slow test.
```bash
[WARNING|utils.py:212] 2024-06-18 16:41:31,610 >> optimum-habana v1.12.0.dev0 has been validated for SynapseAI v1.15.0 but habana-frameworks v1.16.0.526 was found, this could lead to undefined behavior!
[WARNING|utils.py:225] 2024-06-18 16:41:32,355 >> optimum-habana v1.12.0.dev0 has been validated for SynapseAI v1.15.0 but the driver version is v1.16.0, this could lead to undefined behavior!
open_clip_pytorch_model.bin: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 784M/784M [00:02<00:00, 262MB/s]
open_clip_config.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 707/707 [00:00<00:00, 9.98MB/s]
06/18/2024 16:41:35 - INFO - root - Loaded hf-hub:microsoft/BiomedCLIP-PubMedBERT_256-vit_base_patch16_224 model config.
config.json: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 385/385 [00:00<00:00, 3.69MB/s]
06/18/2024 16:41:38 - INFO - root - Loading pretrained hf-hub:microsoft/BiomedCLIP-PubMedBERT_256-vit_base_patch16_224 weights (/root/.cache/huggingface/hub/models--microsoft--BiomedCLIP-PubMedBERT_256-vit_base_patch16_224/snapshots/27005c2e240b3a3a56459db87412a7f3c58abb66/open_clip_pytorch_model.bin).
tokenizer_config.json: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 28.0/28.0 [00:00<00:00, 386kB/s]
vocab.txt: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 225k/225k [00:00<00:00, 3.58MB/s]
============================= HABANA PT BRIDGE CONFIGURATION ===========================
 PT_HPU_LAZY_MODE = 1
 PT_RECIPE_CACHE_PATH =
 PT_CACHE_FOLDER_DELETE = 0
 PT_HPU_RECIPE_CACHE_CONFIG =
 PT_HPU_MAX_COMPOUND_OP_SIZE = 9223372036854775807
 PT_HPU_LAZY_ACC_PAR_MODE = 1
 PT_HPU_ENABLE_REFINE_DYNAMIC_SHAPES = 0
---------------------------: System Configuration :---------------------------
Num CPU Cores : 160
CPU RAM       : 1056374444 KB
------------------------------------------------------------------------------
06/18/2024 16:41:44 - INFO - __main__ - Running warmup
06/18/2024 16:41:46 - INFO - __main__ - Running inference
06/18/2024 16:41:46 - INFO - __main__ - Inference Time per iteration = 5.047ms
06/18/2024 16:41:46 - INFO - __main__ - Throughput = 1.783e+03 images/s
PASSED

==================================================================================================== 2 passed in 208.54s (0:03:28) ====================================================================================================

real    3m29.008s
user    1m53.754s
sys     1m7.392s
```